h/templates/client/thread.html

```diff
@@ -2,8 +2,8 @@
    class="threadexp"
    title="{{vm.collapsed && 'Expand' || 'Collapse'}}"
    ng-click="vm.toggleCollapsed()">
-   <span ng-class="{'h-icon-expand-more': !!vm.collapsed,
-                    'h-icon-expand-less': !vm.collapsed}"></span>
+   <span ng-class="{'h-icon-not-expanded': !!vm.collapsed,
```

Comment:
Are these arrow-drop-down and arrow-drop-up?
https://www.google.com/design/icons/#ic_arrow_drop_down

If so, maybe we don't rename them.